src/main/app/src/hooks/updateActivityLinks.js

```diff
@@ -8,6 +9,7 @@ const updateActivityLinks = (updateLinks, servlet) => {
     // Modify the links that have the data according to the activity (the servlet that is called).
     updateLinks(data);
   }).catch(err => {
+    if (err == "SyntaxError: Unexpected token < in JSON at position 0") updateLinks(tempData);
```

Comment:
I don't like this idea - anyone having the same error in production will have test data populated (which probably doesn't make sense), instead of having a meaningful error message.

Is there a way to make a dummy servlet or something like that?